students/aminavi/session02/grid3.py

```diff
@@ -0,0 +1,7 @@
+def grid(x,y):
+    for i in range(y*2 + 3):
+        if i%(y + 1)==0:
+            print("+" + "-"*x + "+" + "-"*x + "+")
+        else:
+            print("|" + " "*x + "|" + " "*x + "|")
+grid(1,1)
```

Comment:
I'm not sure if you understood this problem correctly? Do you get the same result for like `grid(5,11)`, as shown in the problem description? I don't think so, and I think this code will need to be corrected accordingly. This part can be nontrivial, so please try more.